src/TurnService.ts

```diff
@@ -0,0 +1,71 @@
+import Color from './color';
+
+class TurnService{
+
+  public getThreeColor(side: Color[], sideVals: number[]): any{
+    return sideVals.map((val) => side[val])
+  }
+
+  public setThreeColor(side: Color[], sideVals: number[], newColor: Color[]): any{
+    // needed because we pass newSide colors with default indexes[0,1,2]
+    var count = 0
+
+    sideVals.map((val) =>{ side[val] = newColor[count++] })
```

Comment:
```suggestion
    for (let i = 0; i < sideVals.length; i++) {
        side[i] = newColor[i];
    }
```
or
```
sideVals.map((val, index) => { side[val] = newColor[index] })
```
index will be 0, 1, 2
up to you